Build.cmake

```diff
@@ -101,35 +101,57 @@ ENDIF()
    ENDIF (MSVC OR MINGW)
 
    set(SRC_FILES ${SRC_FILES} ${SRC_PLATFORM_SPECIFIC})
-
-   # Create the g3log library
    INCLUDE_DIRECTORIES(${LOG_SRC})
-  SET(G3LOG_LIBRARY g3logger)
-
-   ADD_LIBRARY(${G3LOG_LIBRARY} SHARED ${SRC_FILES})
-   SET(${G3LOG_LIBRARY}_VERSION_STRING ${VERSION})
-   MESSAGE("Creating ${G3LOG_LIBRARY} VERSION: " ${VERSION})
-   SET_TARGET_PROPERTIES(g3logger PROPERTIES LINKER_LANGUAGE CXX SOVERSION ${VERSION})
-
-   set_target_properties(${G3LOG_LIBRARY} PROPERTIES
+   
+# Create the g3log static library
+   SET(G3LOG_STATIC_LIB g3logger_static)
```

Comment:
Please add an options flag for building it statically.  Please see Options.cmake. 

Also. Why the ```_static``` suffix?